input/src/main/scala/fix/CaseClassBinaryCompatLintingRule.scala

```diff
@@ -0,0 +1,34 @@
+/*
+rule = CaseClassBinaryCompatLintingRule
+ */
+package fix
+
+// format: off
+
+//scalafix:off CaseClassBinaryCompatLintingRule.CaseClassCompanionObject
```

Comment:
I need to turn this off because I cant assert when there are multiple lint errors